packages/firestore/test/unit/model/path.test.ts

```diff
@@ -38,7 +38,7 @@ describe('Path', () => {
   addEqualityMatcher();
 
   it('can be constructed', () => {
-    const path = new ResourcePath(['rooms', 'Eros', 'messages']);
+    const _ = new ResourcePath(['rooms', 'Eros', 'messages']);
```

Comment:
Could you `s/const _ = //` instead? (Or does the linter then complain that you've not used the returned value?)

Reply:
For better of for worse, since we turned of no_unused_expressions, I can indeed just remove the assignment.